lib/http-revocations.js

```diff
@@ -9,6 +9,8 @@ const brZCapStorage = require('bedrock-zcap-storage');
 const {config, util: {BedrockError}} = bedrock;
 const cors = require('cors');
 const helpers = require('./helpers');
+const {validator: validate} = require('./validator');
+const {zcap} = require('../schemas/bedrock-kms-http');
```

Comment:
Recommend using a different name for this var since it could confuse maintainers.

```suggestion
const {zcap: zcapSchema} = require('../schemas/bedrock-kms-http');
```

Reply:
Updated. a6e7734